### PR TITLE
Add icon and text styles

### PR DESCRIPTION
**Pull request by jrmi**

### CODE REVIEW

File: src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardText.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+  { id: 'Press Start', name: 'Press Start' },
+  { id: 'Roboto Mono Medium', name: 'Roboto Mono Medium' },
+  { id: 'Roboto Mono Medium Italic', name: 'Roboto Mono Medium Italic' },
+  { id: 'Sniglet Regular', name: 'Sniglet Regular' },
+  { id: 'Source Code Pro Medium', name: 'Source Code Pro Medium' },
+];
+
+const formatFontValue = val => (val ? val[0] : null);
+const parseFontValue = val => [val];
+const formatFieldValue = val => val?.substring(1, val.length - 1);
```
Comment (dtrucs) [inline src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardText.js:40]: Same thing but shorter 
```js
 const formatFieldValue = val => val?.slice(1, -1);
```

File: src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardIcon.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  const [iconChoices, setIconChoices] = React.useState([]);
+  const { spriteBaseUrl = defaultSpriteBaseUrl } = useSettings();
+
+  React.useState(() => {
+    let mounted = true;
+    const loadIconChoices = async () => {
+      let choices;
+      try {
+        const result = await fetch(`${spriteBaseUrl}.json`);
+        const data = await result.json();
```
Comment (dtrucs) [inline src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardIcon.js:29]: I'm not sure if your code or the following is more optimized but it's more concise with the `then` chaining
```js
const data = await fetch(`${spriteBaseUrl}.json`).then(result => result.json());
```

**Review by dtrucs — DISMISSED** [review]

File: src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardText.js (hunk 1 above)
Comment (jrmi) [reply src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardText.js]: Fixed

File: src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardIcon.js (hunk 2 above)
Comment (jrmi) [reply src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardIcon.js]: I think perf are exactly the same. I propose another more concise and consistent version:

```js
const data = await (await fetch(`${spriteBaseUrl}.json`)).json();
```

**Review by dtrucs — APPROVED** [review]

File: src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardText.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+
+import ColorStyleField from './ColorStyleField';
+import SizeStyleField from './SizeStyleField';
+import TextStyleField from './TextStyleField';
+
+import styles from './styles';
+
+const useStyles = makeStyles(styles);
+
+const fontList = [
```
Comment (CFarcy) [inline src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardText.js:15]: Naive question: in a perfect world, shouldn't it be in a config file or served by the back-end? Or is it something component related?

**Review by CFarcy — APPROVED** [review]

**Pull request merged**